docs/getting-started-scalar-manager.md

```diff
@@ -0,0 +1,117 @@
+# Getting Started with Helm Charts (Scalar Manager)
+Scalar Manager is a web application providing a dashboard for users to check the health and paused status of Scalar products.
+The users can pause or unpause Scalar products through Scalar Manager to backup or restore the underlying databases.
+Scalar Manager also embeds Grafana explorers by which the users can review the metrics or logs of the Scalar products.
+
+## Assumption
+This guide assumes that the users are aware of how to deploy Scalar products with the monitoring and logging tools to a Kubernetes cluster.
+If not, please start with [Getting Started with Scalar Helm Charts](./getting-started-scalar-helm-charts.md) before this guide.
+
+## Environment
+In this guide, we will create the `scalar-manager` component in the following diagram.
+
+```
++--------------------------------------------------------------------------------------------------+
```

Comment:
The direction of `Logging` is correct?
I think we need `verbs` between `scalar-manager` and `lock-stack` and `scalar-manager` and `kube-prometheus-stack`.
Also, should `Managing` be `Manage`? (I don't think `ing` is needed)

Reply:
@feeblefakie 
Understood.
I revised the diagram in https://github.com/scalar-labs/helm-charts/pull/109/commits/697fc8e48c8f51044df731cba27bcf0e9691e084

I think we need to update this other getting-started document as well.
I will do in another PR.

Reply:
Revise other getting-started documents in https://github.com/scalar-labs/helm-charts/pull/112